### DISCUSSION TITLE
How do I update Thorium Portable?

### DISCUSSION THREAD

**Discussion by Daoet** [Alex313031/Thorium-Win#201] [Q&A]
I'm on 117.0 something, but I'd like to update to and open the latest version, while preserving my settings such as:
- opening as standard browser setting (but for the latest version)
I'm opening that with a portable version (.zip) I don't wanna use the mini-installer.exe, only portable version.

I already have a precious default profile in my `C:\Users\$USERNAME\AppData\Local\Thorium\User Data\`, and I'm hoping I could run the new version with this (so not that this profile will get overwritten by launching the new executable)

**Comment by Unknown78** [comment]
There's nothing hurts by backing up your data first.
And then try to replace the `BIN` folder with the latest one.

**Comment by cdrivex4** [comment]
..well..i dont know if you meant to but...I have all zips in a folder... then when I extract to folder I take the new folder name and replace it in the shortcut...
SO my shortcut details looks like this...
Target= C:\Users\main\Downloads\THorium\Thorium_SSE3_124.0.6367.218\BIN\thorium.exe --profile-directory="Profile 1"

Start In= C:\Users\main\Downloads\THorium\Thorium_SSE3_121.0.6167.204\BIN


Once the zip is extracted ill replace the folder name from "Thorium_SSE3_124.0.6367.218" to the newest folder...gone through a few versions this way..and if i dont like the new build I use the older folder name in the shortcut...only had 1 certificate issue once where I had to sign back in on an older version as I had already run a newer build....
I

**Comment by sasik007** [comment]
Is it possible to update using chrlauncher? If so, what should be entered in the ini file?

**Reply by Unknown78** [reply]
Read the topic here:
https://github.com/Alex313031/Thorium-Win-AVX2/issues/6

**Reply by sasik007** [reply]
I read it but it didn't help me much, so is it possible or not?

**Reply by Unknown78** [reply]
Not at the moment I guess.

**Comment by bextrene** [comment]
iDon't know about Thorium az it's very slow & buggy on Windows XP compared 2 more recent versions of Supermium, but  since Thorium is based on Supermium what works there probably works 4 Thorium =)  iUse a 'BAT' file 2 mod the location of things, & then just dump the 'user data' from the old thing into the new thing = replacing it. 4 example, from Supremium 124 into 126, & no problems so far = all the extensions work the same, etc.  iRename the program folder 2 'SLVR' & rename supermium.exe to 'slvr.exe' & run this (a text file with 'bat' extension instead) ~>
cmd /k^
 START "" ".\App\SLVR.exe"^
 --crash-dumps-dir="C:\_Cache_(Deletable)\SLVR\CrashDump"^
 --disk-cache-dir="C:\_Cache_(Deletable)\SLVR"^
 --user-data-dir="%~dp0%\YorData"^
 --ungoogled-supermium^
 --no-first-run^
 --no-default-browser-check^
 --disable-machine-id^
 --screen_ai^
 --keep-old-history^
 --compact-ui^
 --classic-omnibox^
 --classic-omnibox-border^
 --omnibox-autocomplete-filtering^
 --enable-incognito-themes^
 --force-search-engine-choice-screen^
 --scroll-tabs^
 --force-generic-download-icons^
 --old-tab-strip-bounds^
 --double-click-close-tab^
 --legacy-tooltips-win^
 --disable-features=^
PowerBookmarksSidePanel,^
TabHoverCards,^
 --disable-top-sites^
 --allow-running-insecure-content^
 --allow-outdated-plugins^
 --blink-settings=^
mediaControlsEnabled=true,^
requireTransientActivationForGetDisplayMedia=true,^
 --enable-features=^
lazyLoadEnabled=true,^
BlockingDownloadsInAdFrameWithoutUserActivation,^
BlockingFocusWithoutUserActivation,^
EnablePenetratingImageSelection,^
EnsureExistingRendererAlive,^
MinimizeAudioProcessingForUnusedOutput,^
AudioWorkletThreadRealtimePriority,^
 --memory-pressure-off^
 --disable-hang-monitor^
 --disable-logging-redirect^
 --disable-logging^
 --disable-cookie-encryption^
 --disable-encryption^
 --disable-breakpad^
 --disable-crashpad-forwarding^
 --disable-crash-reporter^
 --disable-crashpad-for-testing^
 --disable-crashpad^
 --close-window-with-last-tab=never^

= U can then take that 'YorData' file & dump it into any (so far v126) Supermium version & werx a TREET! =D  No need 2 'sync 2 tha j00gle cloud' or N E other of that (((Bull$hit))), + avoids the deliberate $abotage ware j00gle breaks into yor program & DESTROYS some ov yor extensions, like my 'custom colors' layouts & stuff =P Gotta

**Comment by bextrene** [comment]
LQQX Lyk 'Thorium' fork of Supermium haz bin 'abandoned' =P  Tiz NaMor?  Wachadoin #MrFrik?  N E waze, try Supermium sinz Thorium is 'less supportive' (crashes things) with many more types of pages than Supermium doz, tho it's still a mess with things like Reddit that usually 'fake error out' =P

Frum heer ~>
https://github.com/win32ss/supermium/discussions/1106
"What is this?" = Translation: 'I am pretending iCan't read, because iAm a troll who just wants 2 mock Supermium rather than help enhance it'
_Originally posted by @tumatanquang in https://github.com/win32ss/supermium/discussions/1106#discussioncomment-11830545_

This is the only version of Supermium that should ever exist, & Mr. "Win32" is totally wasting his time focusing on 'Linux' or '64 bit' or other versions, as people can then use 'official' Chrome instead, which supports $treaming $ervices like Nextflix & $camazon right, etc. ~>
https://github.com/win32ss/supermium/releases/download/v126-r6/supermium_126_32_nonsetup.zip
= 32 bit (for Windows XP =) & can launch it as a 'portable', with some switch tweaking.  Here's my current favorite (which evolves like several times a day LOL) ~>
Note: Rename program folder to 'App' &  'supermium.exe' to 'SLVR.exe', & dump this text into 'notepad' or whatever & save it as a 'BAT' file.

cmd /k^
 START "" ".\App\SLVR.exe"^
 --crash-dumps-dir="C:\_Cache_(Deletable)\SLVR\CrashDump"^
 --disk-cache-dir="C:\_Cache_(Deletable)\SLVR"^
 --user-data-dir="%~dp0%\YorData"^
 --blink-settings=^
mediaControlsEnabled=true,^
requireTransientActivationForGetDisplayMedia=true,^
 --enable-features=^
ConsumeCodeCacheOffThread,^
lazyLoadEnabled=true,^
BlockingDownloadsInAdFrameWithoutUserActivation,^
BlockingFocusWithoutUserActivation,^
EnablePenetratingImageSelection,^
MinimizeAudioProcessingForUnusedOutput,^
AudioWorkletThreadRealtimePriority,^
EnsureExistingRendererAlive,^
 --keep-old-history^
 --compact-ui^
 --classic-omnibox^
 --classic-omnibox-border^
 --omnibox-autocomplete-filtering^
 --enable-incognito-themes^
 --scroll-tabs^
 --force-generic-download-icons^
 --old-tab-strip-bounds^
 --double-click-close-tab^
 --legacy-tooltips-win^
 --ash-no-nudges^
 --disable-features=^
PowerBookmarksSidePanel,^
TabHoverCards,^
 --close-window-with-last-tab=never^
 --no-first-run^
 --disable-search-engine-choice-screen^
 --no-default-browser-check^
 --disable-web-security^
 --disable-machine-id^
 --disable-cookie-encryption^
 --disable-encryption^
 --screen_ai^
 --no-pings^
 --disable-top-sites^
 --disable-fetching-hints-at-navigation-start^
 --allow-running-insecure-content^
 --memory-pressure-off^
 --site-per-process^
 --disable-back-forward-cache^
 --disable-auto-reload^
 --disable-hang-monitor^
 --metrics-recording-only^
 --allow-outdated-plugins^
 --disable-component-update^
 --disable-domain-reliability^
 --disable-background-networking^
 --ungoogled-supermium^

**Comment by bextrene** [comment]
BTW: An important note on the '--disable-web-security' switch above = it's obviously the EXACT OPPOSITE!  On Window $ex Pee (XP) '(((Chr0me)))', like so many other ways j00gle deliberately $abotages it, eats up mega RAM & tries 2 even throw it all 2gether in shared 'thread processes', witch wen U hit like around 1.3 gigs tend 2 crash out =P  Ov korz, the fake excuse is always 'for your safety & security'... we are making the browse even more RAM-wasting & unstable, so U R forced 2 close more tabs & 'just google it' (2 C mor adz)... '4 yor $afety&$ecurity ov korz' =P  They R trolling U all with this.

N E waze, the 'default' is 2 bunch a lot of things 2gether into 'shared' processes, which is of course LESS secure than the 'disable security' feature that separates them, 4 the most part, so less likely 2 top out past 1.2 gigs or whatever per process.  Instead they tend 2 stay well under that, even on RAM hogging #CrippleWareJavaHell $ites like j00Toob, witch thay alzo own =P  Hav a LQQK how thay waste like a million characters just 2 play a fuking video ~>
https://justpaste.it/czkvf
All it really takes 2 play it is a simple link, like 'youtube.com/140000000000.webm
= LOL!! =))  That's all the 14 billion videos on Youtube, with 10 'version' options per file (unnecessary), + another 860 billion 'spaces' left for 'expansion', & then could move to mixing in the alphabet as characters, so really it could B a lot shorter than that, & really all Youtube videos should be in vp9 ACTUAL 720p (not 'compressed 2 hell & back'), & a small 240p vp9, & maybe a 480p MP4 = 'good enough' 4 people who 'don't care' about quality, but also as good or better than any DVD movie (frame size wise), so 'perfectly acceptable' for ANY system =)  AV1 is a $kam ~>
https://justpaste.it/ei1zx
N E waze, with just those 3 formats, in QUALITY 'bit rate' rather than like 1/4 what it should be (full of lego blockies) they could save like 80% of their storage space (because anything over ACTUAL 720p is a waste of time at any 'normal viewing distance' because the eyes won't really notice), saving them bazillions of dollars.  But ya blabla = U can also drastically improve the responsiveness & speed & stability ov j00Toob with N E 'chrome-based browser' via 'Project Vorapis' ~>
https://vorapis.pages.dev/#/
U want $um $ Mr. Frik 2 roll that in there with some mods?  Ideally in the future stream it through portable versions of PotPlayer &/or Media Player Classic so can run FX 2 enhance (De-Noise, gamma curve correct, de-band, warpsharp, etc.) + DX 2 avoid horizontal screen tearing in Window $ex Pee! =D  U kan reech mi @ 'balkanguy (at) live (dot) com'.  So, 'until iAm (((BANNED!))) again' HAHA...

**Comment by gz83** [comment]
Close discussions that are outdated and inactive.
